js/mercado.js

```diff
@@ -350,10 +347,65 @@ module.exports = class mercado extends Exchange {
         };
     }
 
+    parseOHLCV (ohlcv, market = undefined, timeframe = '1m', since = undefined, limit = undefined) {
+        return [
+            ohlcv['timestamp'] * 1000,
+            parseFloat (ohlcv['open']),
+            parseFloat (ohlcv['high']),
+            parseFloat (ohlcv['low']),
+            parseFloat (ohlcv['close']),
+            parseFloat (ohlcv['volume']),
+        ];
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'precision': this.timeframes[timeframe],
+            'coin': market.id.toLowerCase (),
+        };
+        if (since !== undefined) {
+            request['from'] = parseInt (since / 1000);
+            if (limit !== undefined) {
+                request['to'] = (this.sum (request['from'], limit * this.parseTimeframe (timeframe)));
+            } else {
+                request['to'] = this.sum (this.seconds (), 1);
+            }
+        }
+        let response = await this.v4PublicGetCoinCandle (this.extend (request, params));
+        return this.parseOHLCVs (response.candles, market, timeframe, since, limit);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined)
+            throw new ArgumentsRequired (this.id + ' fetchOrders () requires a symbol argument');
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let response = undefined;
+        response = await this.privatePostListOrders (this.extend ({
+            'coin_pair': market['base'],
+        }, params));
+        let orders = response['response_data']['orders'];
+        for (let i = 0; i < orders.length; i++) {
+            orders[i] = this.parseOrder (orders[i]);
+        }
+        return orders;
```

Comment:
We will also need to change this loop:

```JavaScript
        let orders = response['response_data']['orders'];
        for (let i = 0; i < orders.length; i++) {
            orders[i] = this.parseOrder (orders[i]);
        }
        return orders;
```

↓

```JavaScript
        const responseData = this.safeValue (response, 'response_data', {});
        const orders = this.safeValue (responseData, 'orders', []);
        return this.parseOrders (orders, market, since, limit);
```